docs/docker.md

```diff
@@ -0,0 +1,142 @@
+
+The app does have a database for identity management, access control, and caching which can be hosted in either MySQL or SQLite.
+
+- [PowerDNS-Admin GitHub](https://github.com/ngoduykhanh/PowerDNS-Admin)
+- [PowerDNS-Admin Settings](https://github.com/ngoduykhanh/PowerDNS-Admin/blob/master/docs/settings.md)
+- [PowerDNS-Admin Wiki](https://github.com/ngoduykhanh/PowerDNS-Admin/wiki)
+
+## Quick reference
+
+- **Maintained by:** [Matt Scott](https://github.com/AzorianSolutions)
+- **Github:** [https://github.com/AzorianSolutions](https://github.com/AzorianSolutions)
+- **Website:** [https://azorian.solutions](https://azorian.solutions)
+
+## TL;DR
+
+    docker run -d -p 8080:80 -e PDAC_SIGNUP_ENABLED=true ngoduykhanh/powerdns-nameserver
+
+## [Azorian Solutions](https://azorian.solutions) Docker image strategy
+
+The goal of creating this image and others alike is to provide a fairly uniform and turn-key implementation for a chosen set of products and solutions. By compiling the server binaries from source code, a greater chain of security is maintained by eliminating unnecessary trusts. This approach also helps assure support of specific features that may otherwise vary from distribution to distribution. A secondary goal of creating this image and others alike is to provide at least two Linux distribution options for any supported product or solution which is why you will often see tags for both Alpine Linux and Debian Linux.
+
+All documentation will be written with the assumption that you are already reasonably familiar with this ecosystem. This includes container concepts, the Docker ecosystem, and more specifically the product or solution that you are deploying. Simply put, I won't be fluffing the docs with content explaining every detail of what is presented.
+
+## Supported tags
+
+\* denotes an image that is planned but has not yet been released.
+
+### Alpine Linux
+
+- main, latest
+- 0.2.4-alpine, alpine, 0.2.4
+
+### Debian Linux
+
+- 0.2.4-debian, debian
+
+## Deploying this image
+
+### App configuration
+
+Configuration of the PowerDNS Admin app may be achieved through a mixture of two approaches. With either approach you choose, you will need to be aware of the various settings available for the app.
+
+[PowerDNS Admin Settings](https://github.com/ngoduykhanh/PowerDNS-Admin/blob/master/docs/settings.md)
+
+#### Approach #1
+
+You may pass app settings as environment variables to the container. These environment variables will be automatically inserted into the app.config object. Any environment variable that begins with PDA_ or PDAC_ and is entirely capitalized, will be inserted into the app.config object.
+
+**One key difference** between the two variable prefixes is that variables prefixed with PDAC_ will have the PDAC_ prefix removed before being inserted into the app.config object.
```

Comment:
Can you elaborate ? Where are both variables used ? It is still confusing for me, I guess it will be for many users.

Maybe a concrete use case example of both to illustrate can also be welcome.

Reply:
I have re-written this section to better address this problem and provide a clear path moving forward. It does contain some notes that if agreed upon, should probably be also copied to a higher level location where you wish to enforce standards.